src/app/inventory/ConnectedInventoryItem.tsx

```diff
@@ -39,6 +40,7 @@ function mapStateToProps(state: RootState, props: ProvidedProps): StoreProps {
   const showRating = shouldShowRating(dtrRating);
 
   return {
+    ...props,
```

Comment:
There's no need to mix props into this - they are already added in by `connect`.